dll/apisets/api-ms-win-core-crt-l1-1-0.spec

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by update.py
 
 @ stdcall __isascii() msvcrt.__isascii
-@ stdcall _atoi64() msvcrt._atoi64
+@ cdecl _atoi64() msvcrt._atoi64
```

Comment:
This change is not useful on a forwarder either.

Reply:
More useful than the -ret64 thing.